app/main.ipynb

```diff
@@ -56,33 +57,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "py_array = [0 for _ in range(1001)]\n",
```

Comment:
The Python array `py_array` is created with a length of 1001 instead of the required 1000. Adjust the range to `range(1000)`.